Odata-docs/webapi-8/overview/overview.md

```diff
@@ -0,0 +1,54 @@
+
+Here are some of the features provided by the library:
+- Exposes a metadata endpoint that returns a document describing the API service' model, the data types it defines and the endpoints and capabilities it exposes. This document makes it easier for clients to consume the service. It can also be used by code generation tools to automatically generate service proxy on the clients to facilitate interaction with the service (e.g., [OData CLI](/odata/odatacli/getting-started), [OData Connected Service](/odata/connectedservice/getting-started)), allowing the user to focus on business logic.
+- Convention-based routing: Requests are routed to controllers and actions based on naming and URL conventions. Fine-grained control over routing can be achieved using ASP.NET Core route template attributes - `Route`, `HttpGet`, `HttpPost`, `HttpPut`, `HttpPatch`, `HttpDelete`, and `HttpHead` - also referred to as attribute routing.
+- Handles serialization and deserialization of requests and responses
+- Validation of requests and responses based on the types defined in the OData model
+- Support for batch requests, which allow you to execute multiple operations in a single API request
+- Handles [OData query options](/odata/concepts/queryoptions-overview) out of the box by transforming queries to your LINQ-enabled data source:
+  - `$filter` allows you to filter results based on specific conditions: `/Products?$filter=Price lt 1000 and Category eq 'Electronics'` will only return electronic products whose price is less than 1,000.
+  - `$select` allows you to limit the fields that are included in the response: `/Products?$select=Name,Category,DateCreated` will only include the `Name`, `Category`, `DateCreated` fields in the response
+  - `$expand` allows you to join and include related data in the response: `/OrderItems?$expand=Product` will return order items and include the related product in the response
+  - `$skip` and `$top` allow you to limit the number of records in the response and can be used for client-driven pagination: `/Products$top=10&$skip=20&$orderby=Name` returns the third page of 10 products sorted by name.
+  - `$apply` allows you to apply aggregations: `/Orders?$apply=groupby(Product/Name, aggregate($count as Count))` returns the number of orders per product name.
+
+
+## Why choose OData Web API:
+
+Here are some of the benefits and common use cases for using OData Web API:
+
```

Comment:
You didn't leave an empty line below _Here are some of the features provided by the library:_ on the section above. Depending on whether it looks better with or without the empty line, determine whether to add an empty line on the other section or remove the empty line here - for consistency.

Reply:
They seem to be rendered the same way on the web page. Let me remove the empty line since I have it in fewer places.